app/models/post.rb

```diff
@@ -1,3 +1,8 @@
 class Post < ApplicationRecord
+
   belongs_to :user
+  
+  validates :content, presence: true
```

Comment:
このままでも動作は問題ないですが、リファクタ可能です！
```suggestion
  validates :content, {length: { maximum: 140 }, presence: true}
```

Reply:
ありがとうございます！この書き方はじめて知りました笑